src/Tribe/Meta/Chunker.php

```diff
@@ -34,6 +34,7 @@
  * or filter the `tribe_meta_chunker_post_types` filter.
  */
 class Tribe__Meta__Chunker {
+	public $chunked_keys_option_name = '_tribe_chunker_chunked_keys';
```

Comment:
Does this need to remain public (we have a getter already - we could add a setter if required, too)?